RecoBTag/SoftLepton/python/SoftLeptonByMVAComputers_cff.py

```diff
@@ -45,3 +45,11 @@
     softPFMuonCommon,
     ipSign = cms.string("positive")
 )
+
+from Configuration.Eras.Modifier_run3_common_cff import run3_common
+run3_common.toModify(softPFElectronComputer,
+                     useCondDB = cms.bool(False),
+)
+run3_common.toModify(softPFMuonComputer,
+                     useCondDB = cms.bool(False),
+)
```

Comment:
Just a suggestion, not a strict requirement (coming from BTV Reco):
To apply these changes to all the Computers which might be called somewhere, these modifications could be extended to also touch `positiveSoftPFMuonComputer`,`negativeSoftPFMuonComputer`,`positiveSoftPFElectronComputer`,`positiveSoftPFElectronComputer` in the exact same way. All six producers here use the same PSet, so they could be changed now as well in one go to also set `useCondDB = cms.bool(False)`, even if these additional four producers are not responsible for any errors in the tests currently.
Otherwise the changes look fine.

Reply:
Hi @AnnikaStein thanks for the suggestion!
Sure I can extend the change to the other producers, but I have one question:
all of them are using the same PSets (`softPF[Muon|Electron]Common`), so wouldn't it be easier to modify directly the "Common" PSets? Or that's too risky because the Common ones might be imported somewhere else?

Reply:
Hi @francescobrivio 
I totally agree it would be easier. My initial concern was exactly what you mentioned, with the risk that it might be imported somewhere else. But actually, using the cmssdt lxr and github search, I only found some place in [HLT for Phase-2](https://github.com/cms-sw/cmssw/pull/35342), where this PSet is defined as well, but I found no place where it gets directly imported. From that point of view, it should be safe to follow up with changing the PSet itself in another PR.

Reply:
Hi @tvami
Ok, will do!

Reply:
Thanks!